matchboxdesigngroup/kindling
# Remove Composer

## Changes proposed in this pull request

This PR removes Composer from the theme.

Closes https://app.asana.com/0/1206339640467531/1206685929995910

## Pre-submit checklist

As the author of this pull request, I verify that:

- [x] I have verified that all automated tests pass or have provided a detailed comment about why I am submitting with a failed pipeline.

## Testing

### How to test the changes in this pull request

Follow the steps below to test the changes in this PR.

1. Pull this branch to your local environment.
2. Navigate to the home page to confirm no errors.
3. Navigate to the Dashboard to confirm no errors.

### Code review

As the code reviewer for this pull request, I verify that:

- [ ] The code is free of obvious errors.
- [ ] The code is free of obvious duplication.

Once testing is complete, notify the author of any failed tests and move the task to "Kick back" in Asana or continue with the "merging" steps below.

## Merging

As the individual merging this pull request, I verify that:
